tools/cpp/osx_cc_configure.bzl

```diff
@@ -157,17 +157,19 @@ def configure_osx_toolchain(repository_ctx, cpu_value, overriden_tools):
 
     env = repository_ctx.os.environ
     should_use_xcode = "BAZEL_USE_XCODE_TOOLCHAIN" in env and env["BAZEL_USE_XCODE_TOOLCHAIN"] == "1"
+    skip_xcode_fetch = "BAZEL_SKIP_XCODE_FETCH" in env and env["BAZEL_SKIP_XCODE_FETCH"] == "1"
```

Comment:
How is this expected to be use?  Does it require setting BAZEL_USE_XCODE_TOOLCHAIN?  If so, perhaps add a comment indicating such?